src/main.c

```diff
@@ -27,14 +27,42 @@ gboolean info_noverify, info_dumpcert = FALSE;
 gboolean status_detailed = FALSE;
 gchar *output_format = NULL;
 
+#include <sys/ioctl.h>
+#include <stdio.h>
+#include <unistd.h>
+
+static gchar* get_progress(gint percentage)
```

Comment:
This should be called something like `make_progress_line`.

Reply:
Fixed.